modules/authentication/proc-provisioning-users-from-microsoft-azure-to-the-software-catalog.adoc

```diff
@@ -22,6 +32,10 @@ catalog:
         tenantId: ${AUTH_AZURE_TENANT_ID}
         clientId: ${AUTH_AZURE_CLIENT_ID}
         clientSecret: ${AUTH_AZURE_CLIENT_SECRET}
+        schedule:
```

Comment:
It'd be nice to add a comment about being careful of setting this frequency, especially for large organizations.  Something like the upstream docs [here](https://backstage.io/docs/integrations/azure/org#installation)